.storybook/decorators/intl/intl.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import { IntlProvider, addLocaleData } from 'react-intl';
-import { withKnobs, select } from '@storybook/addon-knobs/react';
+import { select } from '@storybook/addon-knobs/react';
 import en from 'react-intl/locale-data/en';
 import de from 'react-intl/locale-data/de';
 import es from 'react-intl/locale-data/es';
-import * as messages from '../../../i18n';
+import frFR from 'react-intl/locale-data/fr';
+import zhCN from 'react-intl/locale-data/zh';
```

Comment:
this file is (minified) 11kb. If anything, I would chunk these before chunking our translations.

Reply:
I don't follow. The react-intl data does not get bundled, we do that only in app-kit